packages/twenty-server/test/integration/metadata/suites/object-metadata/utils/force-create-one-object-metadata.util.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  LISTING_NAME_PLURAL,
+  LISTING_NAME_SINGULAR,
+} from 'test/integration/metadata/suites/object-metadata/constants/test-object-names.constant';
+import { CreateOneObjectFactoryInput } from 'test/integration/metadata/suites/object-metadata/utils/create-one-object-metadata-query-factory.util';
+import { createOneObjectMetadata } from 'test/integration/metadata/suites/object-metadata/utils/create-one-object-metadata.util';
+import { deleteOneObjectMetadata } from 'test/integration/metadata/suites/object-metadata/utils/delete-one-object-metadata.util';
+import { findManyObjectMetadata } from 'test/integration/metadata/suites/object-metadata/utils/find-many-object-metadata.util';
+import { isDefined } from 'twenty-shared/utils';
+
+export const forceCreateOneObjectMetadata = async ({
```

Comment:
why not!
Maybe we should make our test fail fast as test output is inaccurate as soon as one test fail anyway.

Reply:
I feel like this is more a local integration tests building tool that you wanna use when creating a test suite and don't wanna have to reset database each time a test fail without a correct cleanup ( even if it should ! )

I won't push its usage to main through my integ tests